src/ert/resources/workflows/jobs/internal-gui/scripts/csv_export.py

```diff
@@ -22,9 +23,9 @@ class CSVExportJob(ErtScript):
 
     Optional arguments:
 
-    ensemble_list: a comma separated list of ensembles to export (no spaces allowed)
-               if no list is provided the current ensemble is exported
-               a single * can be used to export all ensembles
+    ensemble_list: a JSON string representation of a dictionary where keys are
```

Comment:
`ensemble_dict`?

Reply:
Thanks, I opted to use `ensemble_data_as_json`.